lua/wikis/marvelrivals/Infobox/Person/Player/Custom.lua

```diff
@@ -0,0 +1,188 @@
+---
+-- @Liquipedia
+-- wiki=marvelrivals
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local CharacterIcon = require('Module:CharacterIcon')
+local CharacterNames = mw.loadData('Module:CharacterNames')

```

Comment:
imho Lua.import instead of require/mw.loadData the char modules
(also: move them down to the lua imports)